src/components/charts/line-chart.tsx

```diff
@@ -167,7 +167,10 @@ export class LineChart extends BaseComponent<ILineProps, ILineState> {
             minRotation: chartData.dataLabelRotation,
             maxRotation: chartData.dataLabelRotation,
             userCallback: (label: any) => {
-              return Math.ceil(label);
+              return (label === minMaxValues.minA1
+                      ? Math.floor(label)
+                      : Math.ceil(label)
+              );
```

Comment:
This rounds down the min value to prevent duplicate values on x axis
![image](https://user-images.githubusercontent.com/5126913/79151810-fd2b1880-7d7f-11ea-98f1-424091f32c17.png)